modal/image.py

```diff
@@ -465,6 +467,15 @@ async def join():
         obj.force_build = force_build
         return obj
 
+    async def build(self, app: "_App", client: Optional[_Client] = None) -> None:
+        """Build the image immediately rather than waiting for it to be used."""
```

Comment:
Nit but the "waiting for it to be used" here is arguably not a correct description of what usually happens — images get build when we create apps or functions that _will_ use them.